styles/styles.css

```diff
@@ -381,6 +381,75 @@ a:any-link {
   color: #4a7390;
 }
 
+/* wellmark color start */
+/* wellmark color end */
+/* wellmark spacing size start */
+/* wellmark spacing size start */
+form {
+  width: 1275px !important;
+  margin: auto;
+  padding: 114px 0 112px 0;
+}
+@media (min-width: 1380px) {
+  form {
+    width: 1275px !important;
+    margin: auto;
```

Comment:
@MuthuSoundaraPandian form css shouldnt be part of styles now, It should be coming from block